pytorch_to_returnn/torch/nn/functional.py

```diff
@@ -186,6 +186,34 @@ def tensorflow_transpose(input: Tensor, perm: Optional[Union[Dict[int, int], Tup
   return modules.Transpose(perm=perm)(input)
 
 
+def expand(input: Tensor, *sizes: _size):
+  if sizes and isinstance(sizes[0], (tuple, list)):
+    assert len(sizes) == 1
+    shape = tuple(sizes[0])
+  else:
+    shape = sizes
+  assert isinstance(shape, tuple)
+
+  # reshape to add new dims
+  if len(shape) != len(input.shape):
+    assert len(shape) >= len(input.shape), "Cannot reduce dimensions"
+    assert not -1 in [d for d in shape[:-len(input.shape)]], "Size -1 is not allowed for expanded dims"
```

Comment:
```suggestion
    assert -1 not in [d for d in shape[:-len(input.shape)]], "Size -1 is not allowed for expanded dims"
```
This is canonical Python code (PEP8).
(PyCharm should also tell you that, right?)